### ISSUE TITLE
Destructuring

### ISSUE THREAD

**Issue by rwaldron** [leobalter/goiabada#5]
There are probably other examples, but I spotted this one and figured it was just as good as any to use as an example: 

(https://github.com/leobalter/goiabada/blob/master/lib/goiabada.js#L24-L29)

``` js
var assert,
    running = this.running,
    logger = this.logger;
```

Could be written as: 

``` js
var assert,
    { running, logger } = this;
```

**Comment by leobalter** [comment]
that's amazing, I didn't know this.

**Comment by leobalter** [comment]
Fixed at bee13cc
